### DISCUSSION TITLE
Cast mealie recipe to google hub

### DISCUSSION THREAD

**Discussion by skank01** [mealie-recipes/mealie#4565] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Im looking for this too.
I come from a long way.
I use mealie as an addon in HA
First thing to do , before able to cast it, it had to be https
Mealie in home assistant is http

So i had to create reverse proxy using nginx manager... Then create a subdomain in cloudflare, and setup a reverse proxy in nginx manager + ssl certificate
Then i had to open 2 ports on my unifi setup
Now i can reach it from https
Next i created a button in home assistant to start a script where i start casting to a google home hub in my kitchen
This works
I can now see mealie on my hub, + i can even scroll in mealie
However
Before getting in mealie, i need to log in mealie
And there im stuck.
Cause when i set my cursor in my user field (or even password field) , i dont get a keyboard to type, so i cant log in
Stuck here
Really looking forward for a solution for this

Casting from phone is not a good idea cause you cant scroll on the hub

See https://github.com/mealie-recipes/mealie/discussions/3628 but that is closed but not fixed

### Please provide a concise description of the feature that would resolve your issue.

Casting button

### Please consider and list out some caveats or tradeoffs made in your design decision

None

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by skank01** [comment]
Nobody?

**Reply by anightcap** [reply]
In the same boat as you. Did you ever find a method around this? The keyboard actually does pop up on my Nest Hub to enter username/password, but the keys don't actually work.
